pyproject.toml

```diff
@@ -65,4 +65,14 @@ build-backend = "py_build_cmake.build"
 markers = [
     "nightly",
     "precommit: (deselect with '-m \"precommit\"')",
-]
\ No newline at end of file
+]
+
+[tool.ruff]
+line-length = 119
+
+[tool.ruff.lint]
+ignore = ["C901", "E501", "E741", "W605"]
+select = ["C", "E", "F", "I", "W"]
+
+[tool.ruff.lint.isort]
+lines-after-imports = 2
```

Comment:
The change is nice, but these rules are violated because of other python files you didn't change and the next contributor may break it for the files you touched. It's already hard to distinguish functional and style changes. If you applied it to every python file that would become a nightmare. Move to a separate PR with precommit enforcing this.